src/MissionManager/QGCMapPolygonVisuals.qml

```diff
@@ -306,6 +306,25 @@ Item {
         }
     }
 
+    Component {
+        id: edgeLengthHandleComponent
+
+        MapQuickItem {
+            id:             mapQuickItem
+            anchorPoint.x:  sourceItem.width / 2
+            anchorPoint.y:  sourceItem.height / 2
+            visible:        !_circleMode
+
+            property int vertexIndex
+            property real distance
+
+            sourceItem: Text {
+              text: distance.toFixed(2) + " m"
```

Comment:
In my opinion cm resolution / precision of 2 is a bit overkill. 
Precision of 1 should be prefered.
Its better to not overfreight the UI with numbers.